### ISSUE TITLE
[Concept] : [Core] : Script Migration

### ISSUE THREAD

**Issue by h0lybyte** [KBVE/kbve#502]
**Core Concept/Theory**
A clear and concise description of what the concept is. Ex. It would be cool if [...]

There are about 3 folders that contain scripts, we could try to migrate them into the `/tools/` folder and then do a symbolic link for any additional external script that we might need.

- [1] - /tools/ -> /scripts/ -> https://github.com/KBVE/kbve/tree/main/tools/scripts
- [2] - /scripts/ -> https://github.com/KBVE/kbve/tree/main/scripts
- [3] - Public Scripts (TODO) -> https://github.com/KBVE/kbve/tree/main/apps/kbve.com/public/data

It is not a major problem right now but I feel like I could address this right now before it becomes a problem.

Basically remove folder [2] and migrate internal scripts into /tools/scripts/ , while making a public scripts folder for the other scripts within the `/public/data` of the kbve.com.

* * *

**Alternative Ideas**
Is there any other way this concept could be used?

This is just to keep things organized for the future.

* * *

**Alternative Examples/Sources**
Are there any other references that you can provide?

No other examples as of right now.

* * *

**Additional information**
Add any other context or examples of this concept here.

Nothing major right now, just making this issue ticket to help keep track.

**Comment by h0lybyte** [comment]
After a couple of the outpost migrations, I will close this out!

**Comment by h0lybyte** [comment]
The build was successful without having any major issues, thus we can move forward with closing this out.
